back/engines/free/document_annotation/app/models/document_annotation/document_annotation_phase.rb

```diff
@@ -6,7 +6,7 @@ module DocumentAnnotation::DocumentAnnotationPhase
   included do
     with_options if: :document_annotation? do
       validates :document_annotation_embed_url, presence: true, format: {
-        with: %r{\Ahttps://(?:.*\.konveio\.com|.*\.konveio\.site)/.*\z},
+        with: %r{\Ahttps://(?:.*\.konveio\.(com|site|net))/.*\z},
```

Comment:
I'm pretty sure we can drop the last part of the regex, but tomato-TOMATO.
```suggestion
        with: %r{\Ahttps://(?:.*\.konveio\.(com|site|net))/},
```

Reply:
Gonna keep it as is.